handlers/amazon_s3_file_delete_v2/handler/init.rb

```diff
@@ -20,40 +20,46 @@ def initialize(input)
       @parameters[node.attribute('name').value] = node.text.to_s
     end
 
-    @enable_debug_logging = @info_values['enable_debug_logging'].downcase == 'yes' ||
-                            @info_values['enable_debug_logging'].downcase == 'true'
+    @enable_debug_logging = 
+      @info_values['enable_debug_logging'].downcase == 'yes' || 
+      @info_values['enable_debug_logging'].downcase == 'true'
     puts "Parameters: #{@parameters.inspect}" if @enable_debug_logging
 
-    @region = !@info_values["region"].to_s.empty? ? @info_values["region"] : @parameters["region"]
+    @region = !@info_values["region"].to_s.empty? ? 
+      @info_values["region"] : 
+      @parameters["region"]

```

Comment:
Shouldn't we be using the @parameters["region"] value if it is provided? I believe this logic is wrong. It is checking the @info_values["region"] but should be checking the @parameters["region"] value.